apps/nextjs-website/src/lib/api.ts

```diff
@@ -128,3 +131,14 @@ export async function getTutorialLists(productSlug?: string) {
     ) || null;
   return manageUndefinedAndAddProduct(props);
 }
+
+export async function getWebinars(): Promise<readonly Webinar[]> {
```

Comment:
you could even use a single function that accept an optional filter parameter.
e.g. `getWebinars(filter: FilterType) => ...` where filter type could be anything we want, like (`{ startDate: Date, ... }`).
Maybe we can change this later

Reply:
Yes, we will change all these functions once the logic about webinars is established